apps/gaia/src/pages/api/image/index.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next"
+import { v4 as uuidv4 } from "uuid"
+import { IncomingForm } from "formidable"
+import auth0 from "../../../utils/auth0"
+import { insertImage } from "../../../db"
+import minioClient from "../../../utils/minio"
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+}
+
+const handlePost = async (res: NextApiResponse, req: NextApiRequest) => {
+  const { schoolId } = req.query
+  const { studentId } = req.query
+  const imageId = uuidv4()
```

Comment:
Might be better if we extract the "upload file to minio" part into its own function inside minio.ts. We can isolate the env variables and internal minio sdk apis into minio.ts.